docker/docker-compose-hq.yml

```diff
@@ -10,6 +10,7 @@ web:
     - "8000:8000"
   environment:
     PYTHONUSERBASE: /vendor
+    CUSTOMSETTINGS: "docker.localsettings-docker"  
```

Comment:
Okay, fair enough - how about letting the bootstrap copy the docker.localsettings-docker file, if there is no localsettings.py?